tests/unit/steps/test_steps_plan.py

```diff
@@ -758,3 +758,60 @@ async def test_get_upgradable_hypervisors_machines(
     assert {
         hypervisor.machine_id for hypervisor in hypervisors_possible_to_upgrade
     } == expected_result
+
+
+@pytest.mark.parametrize("backup", [True, False])
+@patch("cou.steps.plan.UpgradePlan")
```

Comment:
You do not test what kind of step are in common, only chceling that there are 2 or one step.

Reply:
I've changed a little bit. See if the approach now is better